config/vufind/config.ini

```diff
@@ -2011,6 +2026,8 @@ limits["delete"] = 100
 RefWorks = "record,bulk"
 EndNote = "record,bulk"
 EndNoteWeb = "record,bulk"
+; Note: ZoteroWebLibrary requires proper settings in [Zotero] section!
+ZoteroWebLibrary = "record,bulk"
```

Comment:
Should we comment this out by default since it won't work without specific configuration?